packages/schemas/tables/settings.sql

```diff
@@ -0,0 +1,6 @@
+create table settings (
+    id varchar(128) not null,
+    custom_domain text,
```

Comment:
Since `custom_domain` is not null, would you mind create a database seed issue on Linear to set a value on installation?

Reply:
Issue added. (LOG-1294)